### ISSUE TITLE
Recent versions of Safari can download the result of scrape

### ISSUE THREAD

**Issue by quark67** [philbot9/youtube-comment-scraper#7]
The message "Warning for Safari users" is displayed, but my recent version of Safari (currently, 11.1.1) can download the result of scrape. You could probably remove material added in https://github.com/philbot9/youtube-comment-scraper/commit/0f63507e359f1f86451d1e13f154d113c1b46514 or do a better recognition of old Safari versions (I don't known from which version Safari can download the result of scrape).

**Comment by philbot9** [comment]
Yes, it appears to work on Desktop Safari. On iOS it still doesn't work from what I can tell. We'll have to find a way to distinguish between the two.
